### PR TITLE
Updates to DNS instructions to make it clearer what is needed

### PR DESCRIPTION
**Pull request by hfloyd**
Updates to DNS instructions to make it clearer what is needed. (In response to DNS updates required by new Umbraco Cloud infrastructure)

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @hfloyd !

I'll run it by our Cloud team for a review.
Some of what you've added contradicts the warning we have, about how we only recommend using A-records on the static IPs when using CNAME is not possible - this is simply due to the fact that the IPs are highly volatile to any changes in our Cloud infrastructure 🤔 Let's hear what they have to say, so we can get this just right!

**Comment by hfloyd** [comment]
Definitely get it verified. I made those edits with Jakob's collaboration. We found that the A record was needed for the dns to work.

**Comment by sofietoft** [comment]
Ah, that's right, yes 😄 I've heard before that figuring out which records to use for what, can be a bit confusing, so it would definitely be great to update our docs to be more specific on the topic!

Our Cloud experts will review the PR next week (unfortunately running a bit low on time this week!) 💪

**Comment by hfloyd** [comment]
Sure 😎 I know Cloud customers are getting messages about updating their DNS right now, which is why I had occasion to visit those docs.

**Comment by hfloyd** [comment]
Hi @sofietoft Any update on this? I'm having to send my edits via email to my confused Cloud clients... It would be nice if I could point them to the "official" doc url... 😢

**Comment by sofietoft** [comment]
I apologies for the delay on this one @hfloyd !

We'll take a closer look into this today.
The reason we haven't been able to give you an answer yet is because we need to make sure what we're recommending in our documentation regarding this, is indeed the best practice based on our current infrastructure!

Hopefully we'll be able to resolve this as soon as possible!

**Comment by hfloyd** [comment]
Thanks for the update, @sofietoft . Get some feedback from the Cloud SWAT team - I'm sure they have been helping people with their DNS updates and have seen what works and what just doesn't. 😁

File: Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ You can bind any hostname to your project environments. Keeping in mind that the
 3. If your site should be accessible without any sub-domain (example: https://mysite.com), also add records for the following:
 	-  A wildcard **CNAME** record : HOST = `*` and ANSWER = `dns.umbraco.io`
 	-  An **A** record : HOST = blank and ANSWER = an IP, either `104.19.191.28` or `104.19.208.28` 
-6. Once all your DNS entries have been made, log in to the Umbraco Cloud Portal and add those hostnames to your site. 
-7. It might take several minutes for the certificate to be issues and the DNS to be propagated, so don't be alarmed if your site doesn't pop up immediately at the new domain when checked in a web browser.
+4. Once all your DNS entries have been made, log in to the Umbraco Cloud Portal and add those hostnames to your site. 
+5. It might take several minutes for the certificate to be issues and the DNS to be propagated, so don't be alarmed if your site doesn't pop up immediately at the new domain when checked in a web browser.
```
Comment (greystate) [inline Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md:33]: Super small typo here: "to be issues" => "to be issued"

Comment (hfloyd) [reply Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md]: thanks

**Comment by sofietoft** [comment]
Hi @hfloyd !

Apologies for the long delay on this one 🙏 
So, since we've moved to a new infrastructure on Umbraco Cloud, this PR might not be relevant anymore.

Would you mind running through it, to check whether you agree with that?

**Comment by sofietoft** [comment]
Hi @hfloyd !

As it's been a while since there has been any activity on this one, I'll be closing it down.
If you still think these docs are not up to speed regarding this topic, please get back to us so we can get it resolved properly.

Thanks again for helping us keeping the docs up to date 💪

**Comment by hfloyd** [comment]
Hi @sofietoft , I did see your message back in April, but I didn't have any new hostnames to set up in order to test whether anything had changed or not. I'll revisit this if I have more information.

**Pull request closed without merging**